runtime/vm/jfr.cpp

```diff
@@ -34,13 +34,31 @@
 
 extern "C" {
 
+typedef struct J9JFRTypeID {
+	jlong id;
+	J9Class *clazz;
+} J9JFRTypeID;
+
 #undef DEBUG
 
 // TODO: allow configureable values
 #define J9JFR_THREAD_BUFFER_SIZE (1024*1024)
 #define J9JFR_GLOBAL_BUFFER_SIZE (10 * J9JFR_THREAD_BUFFER_SIZE)
 #define J9JFR_SAMPLING_RATE 10
 
+/* Value needs to be the same as jdk.jfr.internal.JVM.RESERVED_CLASS_ID_LIMIT. */
+#define RESERVED_CLASS_ID_LIMIT 500
+#define INVALID_TYPE_ID -1
```

Comment:
Suggest a blank line before line 51 to clarify that the comment on line 49 only applies to line 50.